src/scripts.js

```diff
@@ -220,190 +283,29 @@ function sleepCardHandler() {
-//     flipCard(event.target.parentNode, stairsMainCard);
-//   }
-//   if (event.target.classList.contains('sleep-go-back-button')) {
-//     flipCard(event.target.parentNode, sleepMainCard);
-//   }
-// }
-
-// function updateTrendingStairsDays() {
-//   currentUser.findTrendingStairsDays();
-//   trendingStairsPhraseContainer.innerHTML = `<p class='trend-line'>${currentUser.trendingStairsDays[0]}</p>`;
-// }
-
-// function updateTrendingStepDays() {
-//   currentUser.findTrendingStepDays();
-//   trendingStepsPhraseContainer.innerHTML = `<p class='trend-line'>${currentUser.trendingStepDays[0]}</p>`;
-// }
-
-// for (var i = 0; i < dailyOz.length; i++) {
-//   dailyOz[i].innerText = currentUser.addDailyOunces(Object.keys(sortedHydrationDataByDate[i])[0])
-// }
-
-
-
-
-
-// hydrationUserOuncesToday.innerText = hydrationData.find(hydration => {
-//   return hydration.userID === currentUser.id && hydration.date === todayDate;
-// }).numOunces;
-
-// hydrationFriendOuncesToday.innerText = userRepository.calculateAverageDailyWater(todayDate);
-
-// hydrationInfoGlassesToday.innerText = hydrationData.find(hydration => {
-//   return hydration.userID === currentUser.id && hydration.date === todayDate;
-// }).numOunces / 8;
-
-// sleepCalendarHoursAverageWeekly.innerText = currentUser.calculateAverageHoursThisWeek(todayDate);
-
-// sleepCalendarQualityAverageWeekly.innerText = currentUser.calculateAverageQualityThisWeek(todayDate);
-
-// sleepFriendLongestSleeper.innerText = userRepository.users.find(user => {
-//   return currentUser.id === userRepository.getLongestSleepers(todayDate)
-// }).getFirstName();
-
-// sleepFriendWorstSleeper.innerText = userRepository.users.find(user => {
-//   return currentUser.id === userRepository.getWorstSleepers(todayDate)
-// }).getFirstName();
-
-// sleepInfoHoursAverageAlltime.innerText = currentUser.hoursSleptAverage;
-
-// stepsInfoMilesWalkedToday.innerText = currentUser.activityRecord.find(activity => {
-//   return (activity.date === todayDate && activity.userId === currentUser.id)
-// }).calculateMiles(userRepository);
-
-// sleepInfoQualityAverageAlltime.innerText = currentUser.sleepQualityAverage;
-
-// sleepInfoQualityToday.innerText = sleepData.find(sleep => {
-//   return sleep.userID === currentUser.id && sleep.date === todayDate;
-// }).sleepQuality;
-
-// sleepUserHoursToday.innerText = sleepData.find(sleep => {
-//   return sleep.userID === currentUser.id && sleep.date === todayDate;
-// }).hoursSlept;
-
+//// END OF SLEEP //
+
+const mainClickHandler = (e) => {
+  // if click happened on step card section:
+  stepCardHandler();
+  // if click happened on climb card section:
+  climbCardHandler();
+  // if click happened on water card section:
+  waterCardHandler();
+  // if click happened on sleep card section:
+  sleepCardHandler();
+}
+// ^^ discuss if we want this. may need to check out / modify html. wonder if html modification is part of refactoring.
```

Comment:
I could see HTML modding in the future but it's hard to tell when this will be relevant. Will discuss later